pysparkling/tests/test_resolve_filenames.py

```diff
@@ -46,7 +46,7 @@ def __init__(self, name):
 
 def test_local_1():
     filenames = File.resolve_filenames(
-        '{}/*'.format(os.path.dirname(CURRENT_FILE_LOCATION))
```

Comment:
Is this actually PySpark's behavior on Windows?

My assumption was that one always uses a POSIX-path for input to PySpark (always `/`) and that when PySpark loads a local file it will convert to the correct separators for that system. Do you have a system where you can test this?

I am not saying this is implemented here. Windows tests are clearly broken. I just don't know what the correct behavior is.

Reply:
I have a Windows pc. And I have a Debian WSL, so I can test both :-). I'll try this later on.
Cleaning up some pylints now.

Reply:
Weird, but thanks a lot for double-checking this.